joern-cli/frontends/rubysrc2cpg/src/main/scala/io/joern/rubysrc2cpg/astcreation/AstForStatementsCreator.scala

```diff
@@ -270,14 +270,25 @@ trait AstForStatementsCreator {
       methodDefInArgument.addOne(ast)
     }
 
+    val prefixMethods = List(
```

Comment:
Faster lookups with a set
```suggestion
    val prefixMethods = Set(
```